### DISCUSSION TITLE
Access to simple code once presented in circuit board forum

### DISCUSSION THREAD

**Discussion by Zyranivia** [sanni/cartreader#64] [General Chat]
This question does not entirely belong here, but I hope you can still help me with my request. German version below, feel free to answer in either German or English.

Here ([29F032 Flashrom mit Arduino beschreiben](https://circuit-board.de/forum/index.php/Thread/17869-29F032-Flashrom-mit-Arduino-beschreiben/)) you presented a program to write to a 29F032 on a ArduinoMega. Currently this thread links to this repository, but there used to be a link to your code just doing that one task. Would it be possible to get this simpler code and the corresponding circuit diagram? Thanks in advance!

----

Hier ([29F032 Flashrom mit Arduino beschreiben](https://circuit-board.de/forum/index.php/Thread/17869-29F032-Flashrom-mit-Arduino-beschreiben/)) hast du ein Programm vorgestellt, mit dem der 29F032 auf dem ArduinoMega beschrieben werden kann. Aktuell verlinkt der Thread auf dieses Repository, früher gab es einen Link zu deinem Code, der nur diese Aufgabe erledigte. 
Könnte ich diese vereinfachte Programm und den dazugehörigen Schaltplan bitte bekommen? Danke!

**Comment by sanni** [comment]
Here is the Arduino file, not sure though if it still works with a newer Arduino IDE:
[FlashromWriterv2.zip](https://github.com/sanni/cartreader/files/5745074/FlashromWriterv2.zip)

**Reply by Zyranivia** [reply]
Thanks for the prompt answer, it worked out perfectly for my purposes.
